index.ts

```diff
@@ -58,40 +59,40 @@ export async function isRunning(player: PlayerName) {
     }
 }
 
-/**
+/** review name
  * Kills the Spotify desktop if it's running
  */
 export function stopSpotifyIfRunning() {
     return musicCtr.stopPlayer(PlayerName.SpotifyDesktop);
 }
 
-/**
+/** review name
  * Kills the iTunes desktop if it's running
  */
 export function stopItunesIfRunning() {
     return musicCtr.stopPlayer(PlayerName.ItunesDesktop);
 }
 
-/**
+/** review name
  * Launches the Spotify desktop if it's not running
  */
 export function startSpotifyIfNotRunning() {
```

Comment:
I think "startSpotify" is enough (whether it was running or not could be part of the return value???)